src/Stratis.Bitcoin/P2P/Peer/NetworkPeer.cs

```diff
@@ -1247,7 +1358,10 @@ public TimeSpan PingPong(CancellationToken cancellation = default(CancellationTo
                 }
 
                 DateTimeOffset after = DateTimeOffset.UtcNow;
-                return after - before;
+
+                TimeSpan res = after - before;
```

Comment:
what does `res` stand for?

Reply:
this code no longer exists

but in general, when i add logs and when i want to log the result value, sometimes i need to introduce a new variable because you don't want to calculate the result again

like here `return after - before;` to log that you'd need to calculate subtraction again
so instead i introduce result variable which i commonly call `res`, `res` has been in the code base for long time, so that's nothing new, stands for result, just reduces the wrist harm by typing less characters :-)

Reply:
I know it stands for result lol, but I wanted you to give it a more meaningful name :-)
It doesn't exist anymore so no problem.

Reply:
i see, that's another discussion then :-) 
i know what you mean now, my personal preference here is that just as in setter we have this `value` name reserved (for obvious reasons though), then in very most cases, i reserve `res` for the result of the function. this preference of mine may very much be influenced by Pascal language that i used to work in ages ago where there is an implicit Result variable that you set function result to.

so when i see `res` in the code, i know this is going to be the thing to be returned, feels helpful
on the other hand, yes, you are not exactly knowing what is that (e.g. type-wise). but if i compare which one is more, to me it is knowing that that thing is the return value

this is why i (rare exceptions exist) don't use `res` for any calculations, like you would first set it to something, then change it later and then again and only then return it. it is more like when you set `res` to something, there is a chance that this will be returned, it is final in that sense that it won't contain a midstate of a computation

but overall still just a personal preference that you might not agree with, which is fine

Reply:
res doesn't really tell me what it is that this variable holds. 
I know it's a TimeSpan, I know it's the result, ok, but what does it represent?
In this case, naming it to something like `payloadReturnInterval` would be much more helpful.
No reason why we should name variables to be descriptive, apart from when they're the return object.